docs/application/native/api/iot-headless/6.0/group__CAPI__MEDIA__WAV__PLAYER__MODULE.html

```diff
@@ -0,0 +1,392 @@
+   </div>
+   
+  </td>
+  
+  
+  
+ </tr>
+ </tbody>
+</table>
+</div>
+
+<!-- Generated by Doxygen 1.7.6.1 -->
+<script type="text/javascript">
+var searchBox = new SearchBox("searchBox", "search",false,'Search');
+</script>
+<script type="text/javascript" src="dynsections.js"></script>
+  <div id="navrow1" class="tabs">
+    <ul class="tablist">
+      <li><a href="index.html"><span>The&#160;Basics&#160;of&#160;Tizen&#160;Native&#160;API&#160;Reference</span></a></li>
+      <li><a href="modules.html"><span>Native&#160;API&#160;Reference</span></a></li>
+      <li>
+        <div id="MSearchBox" class="MSearchBoxInactive">
+          <div class="left">
+            <form id="FSearchBox" action="search.php" method="get">
+              <img id="MSearchSelect" src="search/mag.png" alt=""/>
+              <input type="text" id="MSearchField" name="query" value="Search" size="20" accesskey="S" 
+                     onfocus="searchBox.OnSearchFieldFocus(true)" 
+                     onblur="searchBox.OnSearchFieldFocus(false)"/>
+            </form>
+          </div><div class="right"></div>
+        </div>
+      </li>
+    </ul>
+  </div>
+</div>
+<div id="side-nav" class="ui-resizable side-nav-resizable">
+  <div id="nav-tree">
+    <div id="nav-tree-contents">
+    </div>
+  </div>
+  <div id="splitbar" style="-moz-user-select:none;" 
+       class="ui-resizable-handle">
+  </div>
+</div>
+<script type="text/javascript">
+  initNavTree('group__CAPI__MEDIA__WAV__PLAYER__MODULE.html','');
+</script>
+<div id="doc-content">
+<div class="header">
+  <div class="summary">
+<a href="#func-members">Functions</a> &#124;
+<a href="#typedef-members">Typedefs</a>  </div>
+  <div class="headertitle">
+<div class="title">WAV Player</div>  </div>
+<div class="ingroups"><a class="el" href="group__CAPI__MEDIA__FRAMEWORK.html">Multimedia</a></div></div><!--header-->
+<div class="contents">
+<hr/><a name="details" id="details"></a><h2> </h2>
+<p>The <a class="el" href="group__CAPI__MEDIA__WAV__PLAYER__MODULE.html">WAV Player</a> API provides functions for playing the waveform audio file format(*.wav). </p>
+<h2><a class="anchor" id="CAPI_MEDIA_WAV_PLAYER_MODULE_HEADER"></a>
+Required Header</h2>
+<p>#include &lt;wav_player.h&gt;</p>
+<h2><a class="anchor" id="CAPI_MEDIA_WAV_PLAYER_OVERVIEW"></a>
+Overview</h2>
+<p>The <a class="el" href="group__CAPI__MEDIA__WAV__PLAYER__MODULE.html">WAV Player</a> API allows you to simply play and stop a wav file. To play a certain wav file, call <a class="el" href="group__CAPI__MEDIA__WAV__PLAYER__MODULE.html#gab09d39761d089045b40dc3be00708cf2" title="Plays a WAV file with stream information of sound-manager.">wav_player_start_new()</a> with a path to the .wav file. When playing a wav file is finished, <a class="el" href="group__CAPI__MEDIA__WAV__PLAYER__MODULE.html#ga0514468a3f3e68a660ef9535d7dcb22d" title="Called when a WAV file has finished playing.">wav_player_playback_completed_cb()</a> will be invoked. </p>
```

Comment:
Wht sound manager has - in between ? 

```suggestion
<p>The <a class="el" href="group__CAPI__MEDIA__WAV__PLAYER__MODULE.html">WAV Player</a> API allows you to simply play and stop a WAV file. To play a certain WAV file, call <a class="el" href="group__CAPI__MEDIA__WAV__PLAYER__MODULE.html#gab09d39761d089045b40dc3be00708cf2" title="Plays a WAV file with stream information of sound-manager.">wav_player_start_new()</a> with a path to the .wav file. When playing a WAV file is finished, <a class="el" href="group__CAPI__MEDIA__WAV__PLAYER__MODULE.html#ga0514468a3f3e68a660ef9535d7dcb22d" title="Called when a WAV file has finished playing.">wav_player_playback_completed_cb()</a> will be invoked. </p>
<table class="memberdecls">